cogs/utility.py

```diff
@@ -292,15 +292,13 @@ async def about(self, ctx):
         )
         embed.set_thumbnail(url=self.bot.user.avatar_url)
 
-        desc = "This is an open source Discord bot that serves as a means for "
-        desc += "members to easily communicate with server administrators in "
-        desc += "an organised manner."
+        desc = "This is an open source Discord bot that has been made by you."
```

Comment:
The about command is made for all members, not just the owner. Claiming it was made by them, is wrong (for almost every user who can use it) and could be confusing on what the actual purpose of the bot is. The description you removed would be more beneficial, as it would explain what the bot was, to any member who used the command.